src-tauri/src/github/release_notes.rs

```diff
@@ -58,14 +59,13 @@ pub async fn get_newest_flightcore_version() -> Result<FlightCoreVersion, String
 #[tauri::command]
 pub async fn check_is_flightcore_outdated() -> Result<bool, String> {
     let newest_flightcore_release = get_newest_flightcore_version().await?;
+    let newest_version = Version::parse(&newest_flightcore_release.tag_name[1..]).unwrap();
```

Comment:
Would it make sense to have a default value here using `unwrap_or(_else)` ?

The function is async and will run in a thread, so any errors would be non fatal, but this could at least be useful when `CARGO_PKG_VERSION` is not proper semver.

Reply:
What would the default value be? `0.0.0`?

I think as it's non fatal it's fine to just be an unwrap. In fact, if it fails I will get notified via crash handler and can address accordingly ^^

Reply:
Hmmm, I think it would just be better to let it go to the crash handler instead.